alfresco-value-assistance-share/pom.xml

```diff
@@ -2,13 +2,43 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>org.orderofthebee</groupId>
+    <groupId>eu.xenit.alfresco</groupId>
     <artifactId>alfresco-value-assistance-share</artifactId>
-    <version>1.8</version>
+    <version>1.10.4-xenit-SNAPSHOT</version>
```

Comment:
Looks good, only remark is that the plugin will take care of this right? it will release (publish) 1.10.4>

Reply:
Yes, the version# will be used from the pom.xml file.